packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -162,6 +162,7 @@ export class DetailsColumnBase extends BaseComponent<IDetailsColumnProps> {
         if (this._root!.current!) {
           this._root!.current!.classList!.remove(classNames.borderAfterDropping);
           this._root!.current!.classList!.remove(classNames.noBorderAfterDropping);
+          this._root!.current!.classList!.add(classNames.transparentBorder);
```

Comment:
We should confirm that existing red lines are matched with whichever implementation we decide on.

There _may_ be a way to preserve styling in both states. Unsure of the scope of that work. Can you advise?

Reply:
I have already modified the implementation, Now the column header which has been dropped will display the outline around it and not the border. Outline is not part of the elements width/height whereas border is, so no more 1px movements anywhere. Additionally I'd recommend using outline at any place where we are just trying to gain focus on elements momentarily, as it would prevent DOM reflow. 

Also at the end of entire drag drop transaction, all columns will have similar styling.